### DISCUSSION TITLE
How to disable "Upload file" confirmation dialog?

### DISCUSSION THREAD

**Discussion by mizzuri** [kingToolbox/WindTerm#660] [Q&A] [answered]
To reproduce:
In Filer, open a text file with an editor. This will download the text file and open it with a chosen editor.
Make some changes and save.
A window dialog pops up to confirm every time the file is updated. (Please see the attached screenshot.)

Is there a way to always choose "Yes" and disable the dialog?

Thanks.

![101](https://user-images.githubusercontent.com/1288437/163517211-32db630e-71d3-4972-bbaf-cd4fbe668fe9.png)

EDIT: I understand that clicking "Yes to All" would do what I want for the session.
But I'm looking for a way to completely silence the confirmation dialog.

**Comment by kingToolbox** [comment, marked as answer]
Sorry, there is no way yet. I will support this in the next version `WindTerm_2.5.0_Prerelease_1`. Please stay tuned, thank you.
